src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -457,24 +320,150 @@ function ChplListingEditUploadPage({ id }) {
                 </CardContent>
               </Card>
             </ListingContext.Provider>
-          </div>
-          <div>
-            { newListing
-              && (
-                <ListingContext.Provider value={newListingState}>
-                  <Card>
-                    <Box className={classes.sectionHeader}>
-                      <Typography className={classes.sectionHeaderText} variant="h2">Additional Information</Typography>
-                    </Box>
-                    <CardContent>
-                      <ChplAdditionalInformation
-                        listing={newListing}
-                      />
-                    </CardContent>
-                  </Card>
-                </ListingContext.Provider>
+          </Box>
+          </Box>
```

Comment:
Something is off with the indentation here. Two closing elements, one right after the other, shouldn't be on the same level of indentation